.github/workflows/deploy_tagged.yml

```diff
@@ -1,87 +0,0 @@
-name: Tagged release deploy
```

Comment:
Leave this in place so that we can still invoke this manually for hotfix releases. 
Alternatively, in a smaller or empty repo, see if you can just leave an empty stub or something to basically make the GitHub Actions tab keep this in the list of usable workflows.